### ISSUE TITLE
Dutch and French documentation

### ISSUE THREAD

**Issue by 0x-r4bbit** [angular-translate/angular-translate#174]
This is for @sgara and @nickjanssen

So first of all, thank you guys for contributing to the project. As you said, you both want to translate the documentation I set up to branches for Dutch and French.

@nickjanssen, you can find the branch for dutch here: https://github.com/PascalPrecht/angular-translate/tree/dutch-docs

And @sgara your branch is here https://github.com/PascalPrecht/angular-translate/tree/french-docs

So, here's a little introduction on how to write the docs and what I've already set up.

As you might know, there's for each language a folder under `docs/content/guide` which covers the guide in a specific language. The english guide is under `docs/content/guide/en` the german guide  is under `docs/content/guide/de` and so on.

I already setup guides for `nl` (dutch) and `fr` (french). The `nl` folder only appears in the `dutch-docs` branch whereas the `fr` folder only appears in the `french-docs` branch.

Currently, you'll find in both, dutch and french, the english guide, since I just copied the english documentation to make it easier for you to translate.

So, how to work on this? It's actually pretty easy. All you have to do, is to go into the ngdoc files and translate the english contents to your language. To view your changes, you have to do the following (this part is the shittiest thing on the whole process, so sorry):

you have to run a bash script called `generate_site.sh`. This script generates a `site` folder which contains EVERYTHING. The landing page and each documentation. Which also means, everytime you run `generate_site.sh` you generate all documentations, not only the one for your language. This could take some time sometimes.

However, once `site` folder is created, you can just start a simple http server within that folder and you'll get the full experience.

After reviewing your changes and stopping the server, you should remove the whole `site` folder again. Otherwise you'll have some recursive copy hell which slows down the whole process.

So in short, things work basically like this:

``` bash
$ bash generate_site.sh && cd site && server

// once finished reviewing

$ cd ../ && rm -rf site
```

That's actually all. :) 

FYI: It makes sense to translate the docs chapter-wise and also commit chapter-wise á la

`git commit -m "docs(guide): translates chapter X to french"` or so.

Once you're done, you can just send a PR from your specific doc branch to one of the branches back.

If there are any questions, please lemme know! Thanks a lot for your help!
